bin/setup

```diff
@@ -15,22 +14,33 @@ def log(message)
   puts "\n== #{message} =="
 end
 
-chdir APP_ROOT do
-  log 'Installing Bundler'
+FileUtils.chdir APP_ROOT do
+  # This script is a way to setup or update your development environment automatically.
+  # This script is idempotent, so that you can run it at anytime and get an expectable outcome.
+  # Add necessary setup steps to this file.
+
+  puts '== Installing dependencies =='ß
```

Comment:
FYI: `ß` is a syntax error that breaks `bin/setup`

Reply:
Good spot @jaysonvirissimo

Reply:
Thanks for pointing that out!